docusaurus-docs/conda-store-ui/tutorials/create-envs.md

```diff
@@ -83,6 +83,21 @@ To install packages published only on [PyPI][pypi] using [`pip`][pip], include a
 
 -->
 
+### Set environment variables
+
+:::note
+This features is in the release *after* conda-store version 2024.1.1.
+
+Currently, only the `CONDA_OVERRIDE_CUDA` environment variable is supported to set the CUDA version for GPU-powered workflows.
```

Comment:
I'd link to the official conda documentation here as well, see:
https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html#overriding-detected-packages